### PR TITLE
token <-> index operation

### PR DESCRIPTION
**Pull request by SoluMilken**
token to index (with unk token)

 - `init`: vocab_size, vocabulary, unk_token, necessary_tokens
 - `fit`:
        (1) check unk token not in sentences
        (2) build limited size vocabulary from sentences + (add unk token + add important_tokens must have)
 - `transform`: map token to index based on vocabulary,
 - `inverse_transform`: index to token
 - `state`: token2index, index2token

### CODE REVIEW

**Comment by stegben** [comment]
Did you forget to add a op?

**Comment by SoluMilken** [comment]
added op token to index
fa9235b

File: strpipe/toolkit/test/test_build_vocabulary.py [hunk 1]
```diff
@@ -42,29 +21,32 @@ def test_build_vocabulary_from_sentences_with_limit_size():
...
+#             ['蛋白質'],
+#         ],
+#     )
+#     assert set(output.keys()) == set(
+#         ['<SOS>', '<EOS>', '<PAD>',
+#          '<UNK>', '喜歡', '蛋白質'],
+#     )
+#     assert set(output.values()) == set(
+#         list(range(6)),
+#     )
```
Comment (stegben) [inline strpipe/toolkit/test/test_build_vocabulary.py]: If it's not necessary, simply delete it.

Comment (SoluMilken) [reply strpipe/toolkit/test/test_build_vocabulary.py]: 8dd916d

**Pull request merged**